setup.py

```diff
@@ -24,6 +24,7 @@ def find_stub_files(name: str) -> List[str]:
     "mypy>=0.790",
     "typing-extensions",
     "django",
+    "django-stubs-ext",
```

Comment:
Couldn't this rather be put as an extra dependency? For example, like so:
```py
setup(
    ...,
    # or another name/way of doing
    extras_require={"full": ["django-stubs-ext"]},
)
```

Then users could run `pip install django-stubs[full]` (or the other way around, have full by default and minimal for people that want the minimal, i.e. no monkey patching)

This would prevent adding additional files and dependencies that are not required.

Reply:
It is a direct dependency, but you can just ignore it completely. It does not get easier that that.

Reply:
i'm not sure the savings of a relatively small package are worth the possibility of library consumers accidentally missing the `[full]` part of the install step and wondering what's wrong